### DISCUSSION TITLE
where are the services???

### DISCUSSION THREAD

**Discussion by Clemens-Ratte-Polle** [ferdium/ferdium-app#1537] [General]
where is the list of supported messengers?
i cant find them here
https://ferdium.org/

**Comment by SpecialAro** [comment]
Hi!

There is currently no page featuring the provided services (nice suggestion that we should add one!)

Nevertheless, by looking at this folder list you can see more or less all the services provided by Ferdium

https://github.com/ferdium/ferdium-recipes/tree/main/recipes
